toolium/behave/env_utils.py

```diff
@@ -240,3 +253,14 @@ def execute_after_feature_steps(self, context):
         """
         if not self.feature_error:
             self.__execute_steps_by_action(context, ACTIONS_AFTER_FEATURE)
+
+    @staticmethod
+    def fail_first_step_precondition_exception(scenario):
+        """
+        Fail first step in the given Scenario and add exception message for the output.
+        This is needed because xUnit exporter in Behave fails if there are not failed steps.
+        :param scenario: Behave's Scenario
+        """
+        scenario.steps[0].status = Status.failed
```

Comment:
If you use Status.failed value (3) instead of variable, then you can remove _from behave_ import and behave requirement

Reply:
Status value have to be an enum `behave.model_core.Status`. Using other value (for instance, int: 3) an exception is raised when generating the xUnit report.

Reply:
I have removed the requirement, but some additional logic is needed to identify dynamically the version of Behave (status value have changed in the release `1.2.6`. I assume that, if that code is executed, Behave is properly installed as project dependency.

Reply:
👍